### ISSUE TITLE
Focus mode does not work in Safari on iOS 13 or macOS Catalina

### ISSUE THREAD

**Issue by sisungo** [dotnet/try#500]
#### Describe the bug
Work failed.Sometimes it shows a Loading Page.Sometimes it shows an error page:”This service can’t work now.We are processing.”(Chinese to English translation)

#### Did this error occur while using `dotnet try` or online?
- [ ] `dotnet-try`
- [X] online

#### What kind of error was it?
- [ ] User Interface (UI):  
-  [X] Service Error: Can’t try C# online.I tried in China.
- [ ] Other:

#### Please complete the following:
 - OS
    - [ ] Windows 10
    - [X] macOS
    - [ ] Linux (Please specify distro)
    - [X] iOS
    - [ ] Android
 - Browser 
    - [ ] Chrome
    - [ ] Edge
    - [X] Safari

**Comment by sisungo** [comment]
Maybe it is only in China.

**Comment by LadyNaggaga** [comment]
Could you please provide  a screenshot ?  Or provides the steps you went through so, we can replicate it ?

**Comment by hR7N-tNY** [comment]
I believe sisungo is referring to "Focus Mode" being unavailable. I am experiencing the same issue and have attached a screenshot of the same. This occurs when I click "Enter Focus Mode" at the top right hand side of the screen. I am using Safari for my browser and am in the US. Although the error says that Focus Mode is "temporarily unavailable" this has been going on for at least a week.

<img width="1280" alt="Screen Shot 2019-10-08 at 11 58 07 PM" src="https://user-images.githubusercontent.com/56334270/66453249-5610a180-ea29-11e9-8c0d-f7e96638f49a.png">

**Comment by LadyNaggaga** [comment]
Yes, this is know issue due privacy features in Safari.  You can read this https://github.com/dotnet/try/issues/13 for more details and a workaround. 

If you happen to have Chrome or firefox on your machine it will work just fine. 
![image](https://user-images.githubusercontent.com/2546640/66485676-3607cf00-ea77-11e9-8135-e070aa320034.png)

**Comment by jonsequitur** [comment]
The root cause is different from #13, which was fixed last year. I confirmed Focus Mode is working in Safari on macOS Mojave and confirmed it's not working on iOS 13. I wonder if iOS 13 introduced the issue.

@sisungo and @hR7N-tNY, which version of iOS are you seeing this issue in?

Also, @LadyNaggaga I think you said you upgraded to macOS Catalina? Can you repro the issue there?

**Comment by hR7N-tNY** [comment]
@jonsequitur, I am not using iOS. I'm using macOS Catalina. I can confirm, per @LadyNaggaga's recommendation, that "Focus Mode" does work in Chrome, just not in Safari.

**Comment by LadyNaggaga** [comment]
I was able to reproduce the issue above on Catalina.  
On docs you see this 
![image](https://user-images.githubusercontent.com/2546640/66496159-7a4f9b00-ea88-11e9-906f-437f842bf9b0.png) 
On try.dot.net - When you hit run there is no output 
![image](https://user-images.githubusercontent.com/2546640/66496438-e92cf400-ea88-11e9-85d2-01693cab8698.png)

**Comment by LadyNaggaga** [comment]
https://github.com/dotnet/try/issues/774 - Is a similar
